README.md

```diff
@@ -376,6 +376,16 @@ common_setting: "1"
 !include other_place/include_me.yml
 ```
 
+> **Warning**: You can't have in the same file a top `!include` tag and classic YAML
```

Comment:
Please don't use quoting for things that are not quotes :blush:

Reply:
Rebase done and quoting removed.